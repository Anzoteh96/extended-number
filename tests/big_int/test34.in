-5695662448597411082640283303135525751086343453985117166521020837415111573391967392984123670177585274718
824149814973385277195037650856167434655434955193394188383676798860548505870567214205557955553102457460
