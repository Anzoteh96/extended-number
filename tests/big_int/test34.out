Enter two numbers n and m:
n is -5695662448597411082640283303135525751086343453985117166521020837415111573391967392984123670177585274718
m is 824149814973385277195037650856167434655434955193394188383676798860548505870567214205557955553102457460
n + m is -4871512633624025805445245652279358316430908498791722978137344038554563067521400178778565714624482817258
n - m is -6519812263570796359835320953991693185741778409178511354904697636275660079262534607189681625730687732178
n * m is -4694079153162414875976885985389256779852124215541991182736615947039356463149578345242902573992300654319205864800224514294949987146176748486869328213071730000668931522226616441181176300922211745235008496280
n / m is -6
n % m is -750763558757099419470057397998521143153733722824752036218960044251820538168564107750775936858970529958
